modules/ob-release-notes-1-5-1.adoc

```diff
@@ -0,0 +1,24 @@
+// This module is included in the following assemblies:
+// * about/ob-release-notes.adoc
+
+:_mod-docs-content-type: ASSEMBLY
+[id="ob-release-notes-1-5-1_{context}"]
+= Release notes for {builds-shortname} 1.5.1
+
+{builds-shortname} 1.5.1 is now available on {ocp-product-title} 4.19.
+
+[id="new-features-1-5-1_{context}"]
+== New features
+
+The following sections highlight what is new in {builds-shortname} 1.5.1.
+
+* With this update, {builds-shortname} deployment is FIPS compliant, ensuring that all the cryptographic modules and algorithms meet the security requirements.
+
+[id="fixed-issues-1-5-1_{context}"]
+== Fixed issues
+
+The following list highlights fixed issues in {builds-shortname} 1.5.1.
+
+* Before this update, adding volumes to `Build` and `Buildrun` custom resources(CR) failed due to empty fields in the Custom Resource Definition(CRD). With this update, the empty fields in the CRD are removed.
```

Comment:
I think I forgot to catch it earlier but we should leave a space after resources and the bracket in "custom resources (CR)".